deeptools/bigwigCompare.py

```diff
@@ -93,6 +93,11 @@ def parse_arguments(args=None):
                         'zeros may be wrong and this option should be used ',
                         action='store_true')
 
+    parser.add_argument('--fixedStep',
+                        help='Output in fixed step of bin size (as specified by --binSize), not merging '
```

Comment:
If anyone has a better phrasing here that would be great. I think the PR is fine but the sentences is a bit confusing for me.

Reply:
@WardDeb any idea?

Reply:
Agreed the wording is a bit confusing.
Perhaps:
'Write out all bins (of size --binSize) instead of merging neighbouring bins with equal values.' ?

Reply:
I'll take it along in develop for now and will make the change afterwards.